dts/bindings/video/espressif,esp32_cam.yaml

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (c) 2019, Linaro Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: NXP MCUX CMOS sensor interface
+
+compatible: "espressif,esp32-cam"
+
+include: [base.yaml, pinctrl-device.yaml]
+
+properties:
+  pinctrl-0:
+    required: true
+
+  pinctrl-names:
+    required: true
+
+  sensor:
```

Comment:
I think this is name `source` now, but some drivers might still have it named `sensor` and need to be adapted...
https://github.com/zephyrproject-rtos/zephyr/pull/72623